src/vorta/assets/UI/mainwindow.ui

```diff
@@ -62,13 +62,30 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QToolButton" name="profileAddButton">
+        <property name="text">
+         <string>Add Profile</string>
+        </property>
+        <property name="icon">
+         <iconset>
+          <normaloff>:/icons/plus.svg</normaloff>:/icons/plus.svg</iconset>
+        </property>
+        <property name="iconSize">
+         <size>
+          <width>16</width>
+          <height>16</height>
+         </size>
+        </property>
+       </widget>
+      </item>
       <item>
        <widget class="QToolButton" name="profileRenameButton">
         <property name="toolTip">
          <string>Rename Profile</string>
         </property>
         <property name="icon">
-         <iconset resource="../icons/collection.qrc">
```

Comment:
First time using QT. I'm not sure if this is a problem. The QT creator did this automatically. I assume it is because we now have separate light and dark icon sets.